### ISSUE TITLE
inconsistent return value in ivm::PlatformDelegate::GetForegroundTaskRunner

### ISSUE THREAD

**Issue by zericco** [laverdet/isolated-vm#78]
when building [87a448a434](https://github.com/laverdet/isolated-vm/commit/87a448a434c335a2d0be1d289bf3e05cbe1f0341) under windows with VS2017, i get the following errors:

- platform_delegate.h(95): error C4716: 'ivm::PlatformDelegate::GetForegroundTaskRunner': must
return a value
- platform_delegate.h(91): error C4716: 'ivm::PlatformDelegate::GetBackgroundTaskRunner': must
return a value

```c++
std::shared_ptr<v8::TaskRunner> GetBackgroundTaskRunner(v8::Isolate* /* isolate */) final {
   std::terminate();
}

std::shared_ptr<v8::TaskRunner> GetForegroundTaskRunner(v8::Isolate* /* isolate */) final {
  std::terminate();
}
```

looks like no return value is provided if my C++ is not too rusty.

**Comment by laverdet** [comment]
Thanks for the report, this is fixed and up on npm now as 1.7.1.

**Comment by zericco** [comment]
glad i could help
